contracts/ROSCA.sol

```diff
@@ -6,19 +6,43 @@ pragma solidity ^0.4.4;
  * The winner of the pot is decided by reverse auction (lowest Bid wins).
  *
  * Things still missing/needs fixing:
- * in withdraw() , add fee related logic
- * checkingContribution, check whether or not a member as contributed as least contributionSize at the end of Round
+ * in withdraw() , add fee related logic (done, Sol should still be discussed)
+ *   Sol: will keep deduct a fee, and keep it in contract untill the end?
```

Comment:
those are mainly to jot my memory of how i went about trying/planning to solve the issue